src/components/environment-variables/use-save-env-var.js

```diff
@@ -0,0 +1,28 @@
+import { usePatchJson } from '../../effects';
+import PropTypes from 'prop-types';
+import updatableEnvVar from '../../models/environment-variable/updatable-environment-variable';
+
+const useSaveEnvVar = ({
+  appName,
+  envName,
+  componentName,
+  updatableEnvVars,
```

Comment:
updatableEnvVars is unused

Reply:
I was not sure it should not exist as variable here. Removed.